Controllers/FamilyController.cs

```diff
@@ -139,47 +241,202 @@ public async Task<IActionResult> CreateFamily([FromForm] FamilyViewModel Family)
                 }
             }
         }
-
-        // TODO: send confirmation email
-        var body = await _viewToStringService.ViewToStringAsync("ConfirmationEmail", new { FamilyId = NewFamily.Id });
-        _emailService.SendEmail(NewFamily.Email, "Thanks for signing up for Chefster!", body);
-        
-        var model = new ThankYouViewModel
-        {
-            EmailAddress = User.Claims.FirstOrDefault(c => c.Type == ClaimTypes.Email)?.Value!,
-            GenerationDay = Family.GenerationDay,
-            GenerationTime = Family.GenerationTime
-        };
-
-        return RedirectToAction("ThankYou", "Index", model);
+        return Task.CompletedTask;
     }
 
-    [HttpDelete("{Id}")]
-    public ActionResult DeleteFamily(string Id)
+    private Task UpdateOrCreateMembersAndCreateConsiderations(
```

Comment:
I like that this is a separate function. Could we move this to either the member or consideration services?

Reply:
I wonder if we should create a service that just contains the logic for updating a profile? What are your thoughts there?

Reply:
I think I would prefer to not do that. we have a lot of services as it is, and I think it would be clean to fit it into the services we already have

Reply:
My only push back there is that its logic for both considerations and members so it seems like a code smell to include it in a MemberService or ConsiderationService.

Reply:
now that you put it that way it makes sense where its at, apologies

Reply:
all good! Conversations like this make our code better!